AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -121,6 +121,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 ayedas.com.tr,toroslaredas.com.tr,baskentedas.com.tr##.footer_cerezpolitikasi
 enerjisa.com.tr##.enerjisa-cerez-politikasi-cookie-bar
+latamairlines.com#$#body { overflow: auto !important; }
```

Comment:
Fixed it directly, please take a look at my commit https://github.com/AdguardTeam/AdguardFilters/commit/436ae1866900c1774ddfeac6256f70176136fe83.